docs/ontologymodeling/metadata.md

```diff
@@ -1,3 +1,172 @@
+| In subset - transboundary | http://www.geneontology.org/formats/oboInOwl#inSubset        | Indicate that the term is reported as a “transboundary breed” in DAD-IS                                                                                                        | optional DAD-IS only                                                              |
+| term tracker | http://purl.obolibrary.org/obo/IAO_0000233        | Link to relevant GitHub issue(s) that include discussion and decision on the VBO term.                                                                                                         | optional                                                               |
+
+## VBO ID
+
+- Unique and permanent identifiers
+- IRI format: http://purl.obolibrary.org/obo/VBO_####### (7-digit)
+- OBO ID format: VBO:####### (7-digit)
+
+Example:  VBO:0200406 has the unique ID: http://purl.obolibrary.org/obo/VBO_0200406 
+
+More information on term ID ranges is provided here [link to VBO developer documentation].
+
+
+## VBO Term Label
+
+Term labels are unique and follow the format: 
+- **'Most common name (Species)’**
+in which Species is the English name (e.g. 'Chihuahua (Dog)' (VBO:0200338)). 
+- **'Most common name, Country; (Species)'**
+in which country and species are the English names. (e.g., ‘Jersey Giant, Canada; Chicken’ (VBO:0006068)). This format is used for breeds defined by their country of existence as reported by FAO National Coordinators in DAD-IS.
+
+See more explanation in the [term labels and naming conventions](ontologymodeling/term-labels-naming-conventions.md) section.
+
+## Exact synonym
+
+Any name that is provided by breed sources that represents the same concept as the VBO term is added as an exact synonym. 
+
+Example: “Teckel” is an exact synonym for 'Dachshund (Dog)' (VBO:0200406) 
+
+Notes: 
+- Source(s) for the exact synonym is(are) required. 
+- “Exact synonyms” are optional, except for the one representing the ‘most common name’ of the breed (see below) which is required. 
+
+## Most Common name
+
+'Most common name' is a ‘synonym_type’ annotation property that is added to an ‘exact synonym’ to indicate that the name is the one most commonly used across sources when refering to the VBO term. 
+
+Example, ‘Dachshund’ is the ‘most common name’ for VBO:0200406
+
+## Database cross-reference
+
+- A database cross-reference is an identifier in another ontology, database, or the source that refers to the same concept.
+- Database cross-reference is in the CURIE format
+
+Example: ‘Angus (Cattle)’ (VBO:0000104) has database_cross_reference LBO:0000017. 
+
+The “source” annotation for the database cross reference is the provenance for this mapping between VBO and the other source. Currently, the provenance is the ORCID of the contributor [link to contributor section] who manually curated this mapping.
+
```

Comment:
Let's update all the links later. Just because I don't know the best way to add the links (ie link to the repo or to the document). 
Note that it is the same throughout the document.